### PR TITLE
Move link for incident creation to maintenance incidents index page

### PR DESCRIPTION
**Pull request by bgeuken**
It seems to fit there better than on the general project view.

Before:

After:

### CODE REVIEW

File: src/api/app/views/webui2/webui/projects/maintenance_incidents/index.html.haml [hunk 1]
```diff
@@ -25,6 +25,10 @@
           %th
             Release Targets
       %tbody
+    %li.list-inline-item
```
Comment (DavidKang) [inline src/api/app/views/webui2/webui/projects/maintenance_incidents/index.html.haml]: Missing `ul` tag.

**Comment by coolo** [comment]
I'm fine with that - I'm just worried that you didn't have to touch a rspec for this, which sounds like there is no functional test for this workflow

**Comment by bgeuken** [comment]
For some reason the feature test fails in CircleCI. Though on my local machine it passed. My suspission is that the docker image we use there is not up to date (uses old assets), but I didn't have time yet to verify that.

**Comment by coolo** [comment]
Well, don't you love cassettes? Makes writing test cases so much easier to understand!!

```
[dbbec5f3-175a-460c-89c2-5a77080b8086] [207:33.75] VCR::Errors::UnhandledHTTPRequestError (

================================================================================
An HTTP request has been made that VCR does not know how to handle:
  GET http://backend:5352/build/home:Jane:maintenance_project:0/_result?code=unresolvable&repository=repository_36&view=summary

VCR is currently using the following cassette:
  - /home/frontend/project/src/api/spec/cassettes/Bootstrap_Projects/maintenance_incidents/visiting_the_maintenance_overview.yml
    - :record => :once
    - :match_requests_on => [:method, :uri]

**Comment by coolo** [comment]
I pushed a commit to fix the cassette, hope that's fine

**Comment by bgeuken** [comment]
> Well, don't you love cassettes? Makes writing test cases so much easier to understand!!

Hm, seems I was to sure that it were the assets / js failure.

Thanks!

**Pull request merged**